serverlessworkflow/modules/ROOT/pages/core/understanding-jq-expressions.adoc

```diff
@@ -1 +1,159 @@
-//= Understanding JQ expressions
\ No newline at end of file
+= Understanding JQ expressions
+
+Every workflow instance is associated with a data model. This model regardless of whether your flow file is written in YAML or JSON, consists of a JSON object. The initial content of that object depends on how the flow is started. If the flow is created through a link:{cloud_events_url}[Cloud Event], then the content is taken from `data` property. If started through an HTTP POST invocation, then the request body is expected to have a `workflowdata` property, from where the content of the model is retrieved. 
+
+Expressions are the mechanism defined by the link:{spec_doc_url}#workflow-expressions[Serverless Workflow Specification] for the states to interact with the data model. Kogito supports two expression languages: link:https://github.com/json-path/JsonPath[jsonpath] and link:https://stedolan.github.io/jq/manual/[jq]. The default one is `jq`, but you can change it to `jsonpath` by using the `expressionLang` property. This guide will focus on `jq` and discuss its usage in following uses cases: switch state conditions, action function arguments, and data filtering.
```

Comment:
I do not get whats the point of declaring the link at the beginning if you are going to use it just once.

Reply:
It's the add-hoc standard being requested around the guides, and it's also suggested to use
my_link_bla_bla_url: http://aaaa.bbb.ccc

"_" underscore separation and end with url

Reply:
yes, I know how to use it, but I believe it is useful when you are actually reusing the link (or part of a link) several times, not when you are using the link just once. 
From a maintenance point of view, it is the same to search for link: and update it than search the link at the start of the file. 
This is like we were declaring all strings as constants in our code.

Reply:
@fjtirado, I missed this point on my review. We should use the variables because the content team will use them to update the links when needed. Also, it helps our automation to fix the paths if needed.